bootstrap-themes/pom.xml

```diff
@@ -34,6 +34,24 @@
             <artifactId>wicket-webjars</artifactId>
         </dependency>
 
+        <dependency>
+			<groupId>org.webjars</groupId>
+			<artifactId>bootstrap-italia</artifactId>
+			<version>1.6.2</version>
```

Comment:
Please extract the versions to the `<properties>`. This way it is a bit easier to update them later.